### ISSUE TITLE
Music Player Window Remains in Background when Music Opened [sometimes?]

### ISSUE THREAD

**Issue by robinsoner** [dgventura/gambl#3]
When opening an NSF file via one of the following two methods:
1) Drag the NSF file over the application icon in the dock, or
2) Double click the NSF file
the player window remains behind other windows or simply doesn't 

I'm now having trouble reproducing this.  It happened the first four times I opened the app but now doesn't seem to do it anymore.  Hmmm....

It may be that it doesn't remain in the background but never appeared in the first place.  Not quite sure.  This problem was rectified by clicking the GaMBL icon in the dock.

**Comment by dgventura** [comment]
Yeah, I have seen this sometime too.  Not sure why/what triggers it.  It may be linked to the sometimes launching doesn't start playback automatically bug.
